multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,25 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl
```

Comment:
I agree that it could be useful, but I don't think things like arbitrarily choosing to mount `/srv/kubernetes` is generally useful. I think providing a generically useful kubectl wrapper is something that can be tackled in a separate PR - but I'd rather not have something called `kubectl` in PATH that is very specific in functionality.

As it stands, this is being used exactly one time by this script. I'd probably rather we just execute this in-line or drop the file in /tmp on each boot.